sdk/core/Azure.Core/src/RequestContent.cs

```diff
@@ -17,10 +19,11 @@ namespace Azure.Core
     /// <summary>
     /// Represents the content sent as part of the <see cref="Request"/>.
     /// </summary>
-    public abstract class RequestContent : IDisposable
+    public abstract class RequestContent : BinaryContent
     {
         internal const string SerializationRequiresUnreferencedCode = "This method uses reflection-based serialization which is incompatible with trimming. Try using one of the 'Create' overloads that doesn't wrap a serialized version of an object.";
         private static readonly Encoding s_UTF8NoBomEncoding = new UTF8Encoding(false);
+        private static readonly ModelReaderWriterOptions ModelWriteWireOptions = new("W");
```

Comment:
Can we add a static for the "W" option like [this](https://github.com/Azure/azure-sdk-for-net/blob/9e57c09497c36f8ae981d93c4343fec56c22bf8d/sdk/core/System.ClientModel/src/ModelReaderWriter/ModelReaderWriterOptions.cs#L15)?

Reply:
I believe we decided not to add that to the public API in ClientModel because it was so infrequently used, and there was disagreement around terminology of the word "Wire"

Reply:
Could it be internal then? I'm open to other ideas that would make this "W" option less mysterious.

Reply:
Internal on which type?

Reply:
`internal static ModelReaderWriterOptions Wire => s_wireOptions ??= new ModelReaderWriterOptions("W");`

Reply:
So, make `Wire` an internal property of `RequestContent`?  Let me think about this ... I'm not sure people will think to look for it there, but we might be able to find a logical place for it -- perhaps an internal extension method  to MRWOptions or something similar.